samtranslator/model/apigateway.py

```diff
@@ -132,13 +132,13 @@ def __init__(self, api_logical_id=None, response_parameters=None, response_templ
 
         self.api_logical_id = api_logical_id
         # Defaults to Py27Dict() as these will go into swagger
-        self.response_parameters = response_parameters or Py27Dict()  # type: ignore[no-untyped-call]
-        self.response_templates = response_templates or Py27Dict()  # type: ignore[no-untyped-call]
+        self.response_parameters = response_parameters or Py27Dict()
+        self.response_templates = response_templates or Py27Dict()
         self.status_code = status_code_str
 
     def generate_swagger(self):  # type: ignore[no-untyped-def]
         # Applying Py27Dict here as this goes into swagger
-        swagger = Py27Dict()  # type: ignore[no-untyped-call]
```

Comment:
Nice! Great to see all these removed, and good to see the strict typing has a ripple effect.